docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -21,64 +21,59 @@ tags:
 
 :::tip Support, stability, and dependency info
 
-- In [Public Preview](/evaluate/development-production-features/release-stages#public-preview)
+- API Keys used for Cloud Operations Tools (tcld, API, Terraform) are In [Public Preview](/evaluate/development-production-features/release-stages#public-preview)
+- API Keys used for Namespace Authentication are In [Pre Release](/evaluate/development-production-features/release-stages#pre-release)
 
 :::
 
 Temporal Cloud supports secure programmatic access through API Key Authentication.
 
 ## Overview
 
-Temporal Cloud API Keys facilitate user-level authentication.
-Each key is unique to a single user and ties directly to their Role-Based Access Control (RBAC) settings, ensuring secure and appropriate access.
+Temporal Cloud API Keys provide identity, User and [Service Account](/cloud/service-accounts), based authentication.
+Each key is unique to a single identity and ties directly to their Role-Based Access Control (RBAC) settings, ensuring secure and appropriate access.
 
 The process is as follows:
 
 ```text
-API Key (authentication) → User (identity) → RBAC (authorization)
+API Key (authentication) → Identity (User or Service Account) → RBAC (authorization)
 ```
 
-These API Keys are integral for authenticating Temporal Cloud operator tools, including [Temporal Cloud CLI (tcld)](/cloud/tcld) and the [Temporal Cloud Operations API](/ops).
+API Keys support 2 main use cases:
 
-Users have the autonomy to create, delete, and update access to their API Keys through the Cloud UI or tcld.
+- _**Cloud Operations Automation**_: API Keys can be used as the authentication mechanism for Temporal Cloud operational tools ([Temporal Cloud CLI (tcld)](/cloud/tcld), [the Cloud Ops API](/ops), [Terraform](/production-deployment/cloud/terraform-provider)) to manage your Temporal Cloud account, namespaces, certificates, and user identities.
+  - API Keys for use in Cloud Operations Tools are in Public Preview
+- _**Namespace Authentication**_: API Keys can be used as the Namespace authentication mechanism for executing and managing workflows from the SDK and Temporal CLI as an alternative to using mTLS-based authentication.
+  - API Keys for use in Namespace Authentication are in Pre Release
+  - If you created a Temporal Account using the Self Serve Signup feature, API Keys are available for Namespace Authentication.
```

Comment:
How does a customer know/remember how they signed-up? IIRC, there is no indicator once the Account is created that would tell them this

Reply:
removing these sections per slack convo